finat/sympy2gem.py

```diff
@@ -65,3 +65,72 @@ def sympy2gem_symbol(node, self):
 @sympy2gem.register(symengine.Rational)
 def sympy2gem_rational(node, self):
     return gem.Division(*(map(self, node.as_numer_denom())))
+
+
+@sympy2gem.register(sympy.Abs)
+@sympy2gem.register(symengine.Abs)
+def sympy2gem_abs(node, self):
+    return gem.MathFunction("abs", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.Not)
+@sympy2gem.register(symengine.Not)
+def sympy2gem_not(node, self):
+    return gem.LogicalNot(*map(self, node.args))
+
+
+@sympy2gem.register(sympy.Or)
+@sympy2gem.register(symengine.Or)
+def sympy2gem_or(node, self):
+    return reduce(gem.LogicalOr, map(self, node.args))
+
+
+@sympy2gem.register(sympy.And)
+@sympy2gem.register(symengine.And)
+def sympy2gem_and(node, self):
+    return reduce(gem.LogicalAnd, map(self, node.args))
+
+
+@sympy2gem.register(sympy.Eq)
+@sympy2gem.register(symengine.Eq)
+def sympy2gem_eq(node, self):
+    return gem.Comparison("==", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.Gt)
```

Comment:
`symengine.Gt` and `symengine.Ge`  are not the usual wrapper classes, but `cyfunction`s instead.